.gitignore

```diff
@@ -9,8 +9,9 @@ binaries
 install
 install-debug
 capstone.pc
-version.hh
```

Comment:
`version.hh`  should be in the `.gitignore` file as it contains user specific paths generated at build time